waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/MetaStoreMappingFactoryImpl.java

```diff
@@ -59,15 +64,20 @@ private CloseableThriftHiveMetastoreIface createClient(AbstractMetaStore metaSto
     }
   }
 
+  @SuppressWarnings("resource")
   @Override
   public MetaStoreMapping newInstance(AbstractMetaStore metaStore) {
     LOG
         .info("Mapping databases with name '{}' to metastore: {}", metaStore.getName(),
             metaStore.getRemoteMetaStoreUris());
-    MetaStoreMapping mapping = new MetaStoreMappingImpl(prefixNameFor(metaStore), metaStore.getName(),
+    MetaStoreMappingImpl metaStoreMappingImpl = new MetaStoreMappingImpl(prefixNameFor(metaStore), metaStore.getName(),
```

Comment:
Can we no longer use the Interface type on the LHS here?

Reply:
we can, I'll change it